server/src/database/database.service.ts

```diff
@@ -252,17 +279,38 @@ export class DatabaseService {
     uid: ObjectId,
   ): Promise<void> {
     const region = await this.regionService.findByAppId(appid)
-    const database = await this.findOne(appid)
-    assert(database, 'Database not found')
 
-    const connectionUri = this.getControlConnectionUri(region, database)
+    const sharedDatabase = await this.findOne(appid)
+    const dedicatedDatabase = await this.dedicatedDatabaseService.findOne(appid)
+
+    if (sharedDatabase && dedicatedDatabase) {
+      throw new Error('Database found in both shared and dedicated databases.')
```

Comment:
Database ${appid} 

do that for **elsewhere** the same as it

for being easy to trace logs